### ISSUE TITLE
print or export as pdf

### ISSUE THREAD

**Issue by joonas-yoon** [jwvg0425/algorithm#5]
브라우저에서 인쇄하려고 할 때, 레이아웃이 다 깨져버리네요.

- [ ] print pretty
- [ ] pdf

**Comment by jwvg0425** [comment]
헉 pdf화는 고려를 안 했는데.. collapse 플러그인이 아마 그거 관련해서 정상 동작을 안할 거에요. pdf에서 접는다는 동작이 불가능하기도 하고.. 요쪽 플러그인을 고칠 예정이긴 한데 그 때 한 번 pdf 변환쪽도 같이 확인해보겠습니다

**Comment by joonas-yoon** [comment]
네. 플러그인 문서에서도 아직 TODO에만 있네요. 우선 인쇄하려고 할 때 전부 깨지는 건 이슈 등록이 필요한 내용 같아서 올렸습니다.

**Comment by jwvg0425** [comment]
흠 이게 브라우저에서 인쇄 지원하기가 꽤 까다로워보이는데 별도로 gitbook에서 pdf 파일을 빌드한 다음 이걸 배포하는 건 어떻게 생각하시나요?

**Comment by joonas-yoon** [comment]
안정적이고 좋은 방법이라고 생각합니다.
